WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -99,34 +99,13 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleDismissKeyboardCommand:(FBRouteRequest *)request
 {
-#if TARGET_OS_TV
-  if ([FBKeyboard waitUntilVisibleForApplication:request.session.activeApplication
-                                         timeout:0
-                                           error:nil]) {
-    [[XCUIRemote sharedRemote] pressButton: XCUIRemoteButtonMenu];
-  }
-#else
-  [request.session.activeApplication dismissKeyboard];
```

Comment:
btw, is tapping a key in the keyboard more reliable than `dismissKeyboard` on iPhone as well??
I can test on real devices (iPhone and iPad) later (maybe tonight tho)

Reply:
internally "dismissKeyboard" does exactly the same. And it also fails the test if the button is not found, which we don't need